model/src/main/kotlin/utils/DatabaseUtils.kt

```diff
@@ -36,6 +36,11 @@ import org.ossreviewtoolkit.utils.ort.ORT_FULL_NAME
 import org.ossreviewtoolkit.utils.ort.log
 
 object DatabaseUtils {
+    /**
+     * This map holds the [HikariDataSource] based on the [PostgresConfiguration].
+     */
+    private val dataSources: MutableMap<PostgresConfiguration, Lazy<DataSource>> = mutableMapOf()
```

Comment:
Does this have to be concurrent map, so that it can be used safely from different threads?